views/content-cross-linked.phtml

```diff
@@ -1,3 +1,16 @@
 <?= $data->content ?>
 <hr>
-<p><em>Also published on <a href="<?= $data->medium_post->url ?>">Medium</a>.</em></p>
+<p>
+	<em>
+		<?php
+		printf(
```

Comment:
@majelbstoat Cool man. I probably won't spend the time to go back and reformat all this, but feel free to take it all do as you wish. I'll take your coding style into account next time, it's hard for me not to follow WP coding standards because I have a PHP Code Sniffer lint tool in my IDE that barks at me if I'm a bad boy.

Worth a look https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards